docs/getting-started/using-testnet.md

```diff
@@ -0,0 +1,46 @@
+# Using Testnet Guide
+
+last updated for v1.0.0/testnet2
+
+---
+
+## Why Use Testnet?
+
+Testnet2 is a wonderful place where you can experiment with the Decred applications without worrying that a mistake will cost you real money. It is actually recommended that people use the Testnet2 to learn the basics of the Decred software or any new features.
+
+Decred is currently on its 2nd Testnet, also known as testnet2. Testnets are periodically reset to help keep a manageable blockchain file size. 
+
+---
+
+## How to Run a Testnet Node
+
+Running a testnet2 node is incredibly easy. You application of choice will need to download the testnet2 blockchain, and you will need to create a new wallet file for testnet2 use. Your mainnet blockchain and wallet files will remain untouched. Switching between the two is incredibly easy.
+
+---
+
+## Paymetheus
+
+To launch `Paymetheus` on testnet2, simply launch the `Decred Testnet` application instead of the usual `Decred` application. You will then be walked through the wallet creation (the same steps in the [Paymetheus Setup guide](/getting-started/user-guides/paymetheus.md)). After `dcrd` finishes syncing in the background, you'll then be able to fill your `Paymetheus` wallet with tDCR and familiarize yourself with the software.
```

Comment:
Do we ever call the coins tDCR anywhere else?  Not saying I don't like the idea but I don't think I've ever seen it as anything other than testnet coins elsewhere.